MCP_Server/tools/track_tools.py

```
@@ -0,0 +1,144 @@
+def register(mcp: FastMCP, get_connection, cache):
+    """Register track tools with the MCP server"""
+
+    @mcp.tool()
+    async def create_midi_track(index: int = -1) -> str:
+        """Create a new MIDI track. Use index=-1 to add at the end of the track list."""
+        try:
+            conn = await get_connection()
+            result = await conn.send_command("create_midi_track", {"index": index})
+            cache.invalidate_all()
+            return json.dumps(result, indent=2)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)